storage/innobase/xtrabackup/doc/source/release-notes/2.4/2.4.14.rst

```diff
@@ -0,0 +1,51 @@
+================================================================================
+|Percona XtraBackup| |release|
+================================================================================
+
+Percona is glad to announce the release of |Percona XtraBackup| |release| on
+|date|. Downloads are available from our `download site
+<http://www.percona.com/downloads/XtraBackup/Percona-XtraBackup-2.4.13/>`_ and
+from :ref:`apt <apt_repo>` and :ref:`yum <yum_repo>` repositories. This release
+introduces the suppport of Red Had Enterprise Linux 8.
+
+|Percona XtraBackup| |release| enables saving backups to an Amazon S3 storage
+when using |xbcloud|. The following example demonstrates how to use an Amaxon S3
+storage to make a full backup:
+
+.. code-block:: bash
+
+   $ xtrabackup --backup --stream=xbstream --extra-lsndir=/tmp --target-dir=/tmp | \
+   xbcloud put --storage=s3 \
+   --s3-endpoint='s3.amazonaws.com' \
+   --s3-access-key='YOUR-ACCESSKEYID' \
+   --s3-secret-key='YOUR-SECRETACCESSKEY' \
+   --s3-bucket='mysql_backups'
+   --parallel=10 \
+   ${date}-full_backup
+
```

Comment:
this should be part of xbcloud [docs](https://www.percona.com/doc/percona-xtrabackup/2.4/xbcloud/xbcloud.html)

Reply:
I know. the related pull request will be added shortly based on PXB-1813

Reply:
ok